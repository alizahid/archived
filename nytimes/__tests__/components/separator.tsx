import React from 'react'
import { create } from 'react-test-renderer'

import Separator from '../../src/components/separator'

it('renders', () => {
  const tree = create(<Separator />)

  expect(tree).toMatchSnapshot()
})
