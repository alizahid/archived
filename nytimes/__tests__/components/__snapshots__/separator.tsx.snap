// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<View
  style={
    Object {
      "borderTopColor": "#EFEFF4",
      "borderTopWidth": 1,
    }
  }
/>
`;
